### ISSUE TITLE
Vue.use order for ExpansionPanel and Button breaks expansion panel toggle button

### ISSUE THREAD

**Issue by ludmiloff** [museui/muse-ui#1123]
Hello, first thank you for your efforts on making muse-ui.
I've encountered a little problem:
when 
```
Vue.use(Button); // first
Vue.use(ExpansionPanel); // then
```

the styling of  mu-expansion-toggle-btn is broken causing to not align itself at right;

when
```
Vue.use(ExpansionPanel); // first
Vue.use(Button); // then
```

all is ok.

**Comment by ludmiloff** [comment]
Sorry, I'm asking to reopen this issue again because the fix not working at all.
Here are the details as shown on the screenshot
![screen shot 2018-08-22 at 22 08 39](https://user-images.githubusercontent.com/673583/44485031-0968dc00-a658-11e8-9ba1-aae87c97e6ab.png)

It is obvious the arrow is in the mu-icon-button, not in mu-button.
Please, fix it.
Thanks

**Comment by myronliu347** [comment]
retry, fixed in 3.0.0 
```
yarn add muse-ui
```
